src/main/java/ui/PanelMenuCreator.java

```diff
@@ -48,15 +42,33 @@ public Menu generatePanelMenu() {
         items.add(createLeftPanelMenuItem());
         items.add(createRightPanelMenuItem());
         items.add(closePanelMenuItem());
-        autoCreateItems.add(createCustomizedPanelMenuItem(ASSIGNEE_PANEL_NAME, ASSIGNEE_FILTER_NAME));
-        autoCreateItems.add(createCustomizedPanelMenuItem(MILESTONE_PANEL_NAME, MILESTONE_FILTER_NAME));
-        autoCreateItems.add(createCustomizedPanelMenuItem(UPDATED_PANEL_NAME, UPDATED_FILTER_NAME));
+        for (Map.Entry<String, String> entry :
+                generateCustomizedPanelDetails(prefs.getLastLoginUsername()).entrySet()) {
```

Comment:
This part can be made less verbose. Method names can be shortened (`customPanelDetails`?). If needed, you can also give names to intermediate values.